WordPress/Sniffs/NamingConventions/ValidVariableNameSniff.php

```diff
@@ -60,22 +60,32 @@ class WordPress_Sniffs_NamingConventions_ValidVariableNameSniff extends PHP_Code
 	);
 
 	/**
-	 * Custom list of variables which can have mixed case.
+	 * Custom list of properties which can have mixed case.
 	 *
-	 * @since 0.10.0
+	 * @since 0.11.0
 	 *
 	 * @var string[]
 	 */
-	public $customVariablesWhitelist = array();
+	public $customPropertiesWhitelist = array();
 
 	/**
-	 * Whether the custom whitelisted variables were added to the default list yet.
+	 * Whether the custom whitelisted properties were added to the default list yet.
 	 *
-	 * @since 0.10.0
+	 * @since 0.11.0
```

Comment:
I'd still leave this at `0.10.0`, but may be add a note that the property name was adjusted.